docs/assets/sphinx_rsts/intro/installation/linux.rst

```diff
@@ -1,4 +1,4 @@
-Installing PALISADE on Linux
+Installing OpenFHE on Linux
 ====================================
 
 .. topic:: Quick Note for CentOS CentOS Installation
```

Comment:
I'm inclined to leave it as is mostly because these are just examples. We can remove the numbers and just put `X` there if you'd prefer e.g 

`    First install clang++, e.g., "sudo apt-get install clang-9" to install clang 9.`

would become

`    First install clang++, e.g., "sudo apt-get install clang-X" to install clang version X.`

Reply:
Fair points, and I'd rather leave it with version numbers intact then.